tests/e2e/common.go

```diff
@@ -437,6 +460,103 @@ tail /dev/random;
 	return nil
 }
 
+func (td *OsmTestData) installCertManager(instOpts InstallOSMOpts) error {
+	By("Installing cert-manager")
```

Comment:
somehow related, I'm scared of `common.go` growing with no boundaries. problem of always using _helpers.go_ or _common.go_ or _utils.go_...

Reply:
The file becoming too big is easy to fix, but the harder problem would be controlling the growth of the number of methods on `OsmTestData`.